src/oscar/apps/catalogue/managers.py

```diff
@@ -18,7 +18,16 @@ def base_queryset(self):
 
     def browsable(self):
         """
-        Excludes non-canonical products.
+        Excludes non-canonical products and disabled products
+        """
+        return self.filter(parent=None, is_public=True)
+
+    def browsable_dashboard(self):
+        """
+        Products that should be browsable in the dashboard.
+
+        Excludes non-canonical products, but allows diabled products to edit
```

Comment:
For consistency of documentation please change this to "but includes non-public products". Disabled does not mean the same as "not public".

Reply:
feedback applied